spaceros/Earthfile

```diff
@@ -144,6 +144,28 @@ rosdep:
   RUN sudo rosdep init && rosdep update
   RUN rosdep install --from-paths src --ignore-src --rosdistro rolling -y --skip-keys "console_bridge fastcdr fastrtps rti-connext-dds-5.3.1 urdfdom_headers rmw_connextdds ros_testing rmw_connextdds rmw_fastrtps_cpp rmw_fastrtps_dynamic_cpp composition demo_nodes_py lifecycle rosidl_typesupport_fastrtps_cpp rosidl_typesupport_fastrtps_c ikos"
 
+  RUN sudo apt-get install --yes \
+           gcc g++ cmake libgmp-dev libboost-dev libboost-filesystem-dev \
+           libboost-thread-dev libboost-test-dev \
+           libsqlite3-dev libtbb-dev libz-dev libedit-dev \
+           python3 python3-pip python3-venv \
+           llvm-14 llvm-14-dev llvm-14-tools clang-14
+
+  WORKDIR $SPACEROS_DIR
+  RUN git clone -b v3.2 --depth 1 https://github.com/NASA-SW-VnV/ikos.git
```

Comment:
Imo we should clone it the same way as it is done for cobra, by creating `ikos_vendor` package in `ament_ikos` repo. See: https://github.com/ament/ament_cobra/tree/master/cobra_vendor.

Thoughts?

Reply:
I'm worried that it might take quite a bit to prepare that. Note that the repo is removed right after installation.

I'll also note that we are also looking into using DEBs in the future, so I'm not too worried because this installation procedure is temporary, just to at least have something. (I could provide DEBs now but I don't know where they should be housed, especially since we expect Debian and hopefully also Ubuntu to make IKOS available in their repos.)